plugin-flex-ts-template-v2/src/utils/feature-loader/teams-filters.ts

```diff
@@ -34,6 +36,6 @@ export const init = async (flex: typeof Flex, manager: Flex.Manager) => {
 };
 
 export const addHook = (flex: typeof Flex, manager: Flex.Manager, feature: string, hook: any) => {
-  console.info(`Feature ${feature} registered teams filter hook: %c${hook.teamsFilterHook.name}`, 'font-weight:bold');
+  logger.info(`Feature ${feature} registered teams filter hook: %c${hook.teamsFilterHook.name}`);
```

Comment:
This log has a `%c` for formatting. We need to either remove these or find a way to format console logs that pass through the logger.

Reply:
Let's keep these feature loader logs using `console.info`. It gives us the formatting, plus if there is an issue with bootstrapping then we would want the logs always going to the console anyway.

Reply:
Done!